### PR TITLE
Add python tracer test runner

### PR DESCRIPTION
**Pull request by labbati**
We add the python tracer test runner to the docker library as we do for other integrations.
Previously, we had an image defined into a `Dockerfile` in our repo but for consistency with other integrations we prefer to have it defined here in our docker library repo.

The main objective of this PR is:
 - move the file https://github.com/DataDog/dd-trace-py/blob/168135cdb60ae892addd7b09b30fc71fba772b3a/.circleci/images/runner/Dockerfile to this repo
 - move from an ubuntu based image to an alpine based image to save some space

Please note that a build setting has already been added to https://hub.docker.com/r/datadog/docker-library/~/settings/automated-builds/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ Currently available tags:
 * [`ddtrace_rb_2_2_10`](https://github.com/DataDog/docker-library/tree/master/dd-trace-rb/2.2.10): Test runner for Ruby version 2.2.10
 * [`ddtrace_rb_2_3_7`](https://github.com/DataDog/docker-library/tree/master/dd-trace-rb/2.3.7): Test runner for Ruby version 2.3.7
 * [`ddtrace_rb_2_4_4`](https://github.com/DataDog/docker-library/tree/master/dd-trace-rb/2.4.4): Test runner for Ruby version 2.4.4
+* [`ddtrace_py`](https://github.com/DataDog/docker-library/tree/master/dd-trace-pi): Test runner for Python
```
Comment (masci) [inline README.md]: typo: s/`dd-trace-pi`/`dd-trace-py`

Comment (labbati) [reply README.md]: so close!!! thanks 👍

**Comment by labbati** [comment]
Fixed the typo in the README, available for review again. Thanks!

**Comment by labbati** [comment]
If you mean that I should add the label to the build settings page (https://hub.docker.com/r/datadog/docker-library/~/settings/automated-builds/) that is already done.

**Pull request merged**